tests/build-images.sh

```diff
@@ -1,14 +1,26 @@
 #!/usr/bin/env bash
 set -ex
 
-ver=(NATS_SERVER 2.9.x)
+ver=(NATS_SERVER 2.9.23)
 
 (
-	cd "${ver[1]}/alpine3.18"
+	cd "2.9.x/alpine3.18"
```

Comment:
Should this be 2.9.x here still  Surely this should stay as `${ver[1]}` ?

Reply:
This is the folder which would 2.9.x, an example run is here: https://github.com/nats-io/nats-docker/actions/runs/6502184766/job/17660784118